.buildkite/pipeline.build.yml

```diff
@@ -170,7 +170,7 @@
     - ./java/test.sh
 
 - label: ":cpp: Ray CPP Worker"
-  conditions: [ "RAY_CI_CPP_AFFECTED" ]
+  conditions: [ "RAY_CI_PYTHON_AFFECTED" ]
```

Comment:
This is just to trigger the test, will revert.

Reply:
I think if you do `[all_test]` in the original PRs, you will get that also.

Reply:
Learned something new!!